src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -257,6 +273,86 @@ export default class GeFormRenderer extends NavigationMixin(LightningElement) {
         return invalidFields.length === 0;
     }
 
+    donorTypeInvalid( sectionsList ){
+
+        // aux vars
+        let miniFieldWrapper = {};
+        // get label and value using apiName as key from fields for each section
+        sectionsList.forEach(section => {
+            miniFieldWrapper = { ...miniFieldWrapper, ...(section.getValidationHelper)};
+        });
+
+        // if no donation donor selection, nothing to validate here yet
+        if (isEmpty(miniFieldWrapper[DONATION_DONOR_FIELD_INFO.fieldApiName].value)) {
+            return false;
+        }
+
+        // is donation donor valid
+        return this.getDonorTypeValidationError(miniFieldWrapper);
+
+    }
+
+    getDonorTypeValidationError( fieldWrapper ){
+
+        // relevant donation type fields for validation
+        const DONATION_FIELDS = {
+            account1ImportedField:  ACCOUNT1_IMPORTED_FIELD_INFO.fieldApiName,
+            account1NameField:      ACCOUNT1_NAME_FIELD_INFO.fieldApiName,
+            contact1ImportedField:  CONTACT1_IMPORTED_FIELD_INFO.fieldApiName,
+            contact1LastNameField:  CONTACT1_LASTNAME_FIELD_INFO.fieldApiName,
+            donationDonorField:     DONATION_DONOR_FIELD_INFO.fieldApiName,
+        };
+
+        // relevant Donation_Donor picklist values for validation
+        const DONATION_DONOR = {
```

Comment:
These constants are defined already in utilTemplateBuilder.js

Reply:
Sure, I will grab definitions from there. 
I still fill that working with them as a class is way better than having to strings, so i will do something like:
`const DONATION_DONOR = {
    isAccount1: Account1,
    isContact1: Contact1
}
`
Hope this is ok with you, I think it will help reading the code in the feature